lib/cosmos/dart/processes/dart_ingester.rb

```diff
@@ -16,20 +16,57 @@
 # Handles packets by writing them to the dart log file. New SYSTEM META packets
 # cause a new log file to be started.
 class DartInterfaceThread < Cosmos::InterfaceThread
+  include DartCommon
+  
   attr_writer :packet_log_writer
   attr_writer :log_type
-
+  
   def handle_packet(packet)
     if packet.target_name == 'SYSTEM'.freeze and packet.packet_name == 'META'.freeze
       Cosmos::Logger.info("#{@log_type}: #{packet.target_name} #{packet.packet_name}")
+      
+      # Manually read the configuration from the buffer because the packet might not be identified if
+      # identify_and_define is false
+      buffer = packet.buffer(false)
+      if buffer.length >= 33
+        system_config_name = Cosmos::BinaryAccessor.read(8, 256, :STRING, buffer, :BIG_ENDIAN)
+        # Switch to this system_config
+        begin
+          switch_and_get_system_config(system_config_name)
+        rescue Exception => e
+          Cosmos::Logger.error(e.formatted)
+          Cosmos::Logger.error("Could not load system_config: #{system_config_name}")
```

Comment:
You log the error in this case but if you can't get the system config what happens?

Reply:
This is before logging the raw data as part of ingest.    After the system config log fails, the system is back at the latest configuration it has.   That only affects the ability to pull the correct packet_time from the packet inside of the packet log writer.